scripts/pipelines/build_artifact

```diff
@@ -7,6 +7,7 @@ export PATH=${COMPOSER_BIN}:${PATH}
 # Generate artifact in a separate directory.
 export NEW_BUILD_DIR=/tmp/artifact
 blt artifact:build -D deploy.dir=${NEW_BUILD_DIR} --ansi --verbose
+
```

Comment:
same as above, if this file difference is only 1 line of whitespace then can you revert it please?

Reply:
Also same as above.